Makefile

```diff
@@ -15,15 +26,26 @@ base-docker-run = docker run \
 	-e AZURE_SUBSCRIPTION_ID=$(AZURE_SUBSCRIPTION_ID) \
 	-e AZURE_TENANT=$(AZURE_TENANT_ID) \
 	-e AZURE_TENANT_ID=$(AZURE_TENANT_ID) \
-	-v $(shell pwd):/packer-images
+	-v $(shell pwd):/packer-images \
+	$(docker_ssh_opts) \
+
+guard-%:
```

Comment:
Have you mentioned the necessity of exporting `env="something"` before running terraform make targets?

Reply:
Looks like only `images-builder` and `images-tester` are valid options.  
@lborguetti why do we use GNU Make as our "task runner"?  
I really would like something nicer for reading parameters. Maybe write our CLI in bash/python/go. Don't know what people usually do 🤔

Reply:
It's necessary declare `env=` to execute the targets, but it's not documented yet.